src/databricks/labs/lsql/dashboards.py

```diff
@@ -893,6 +897,35 @@ def _from_dashboard_folder(cls, folder: Path) -> "DashboardMetadata":
             tiles.append(tile)
         return cls(display_name=folder.name, _tiles=tiles)
 
+    def export_to_zipped_csv(self, sql_backend: SqlBackend, export_path: Path) -> Path:
+        """Export the dashboard queries to CSV files directly into a ZIP archive."""
+        zip_export = export_path / "export_to_zipped_csv.zip"
```

Comment:
@jgarciaf106 : Please except the function argument for the `zip_export` instead of adding the hardcoded "export_to_zipped_csv.zip"